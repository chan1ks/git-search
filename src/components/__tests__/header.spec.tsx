import React from 'react';

import renderer from 'react-test-renderer';
import Header from '../__layout/header';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header title="This is a title!" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
