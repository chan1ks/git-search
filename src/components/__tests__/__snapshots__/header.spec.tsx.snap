// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<h1
  style={
    Object {
      "marginBottom": "10px",
    }
  }
>
  This is a title!
</h1>
`;
